app/src/main/java/com/framgia/wsm/screen/confirmrequestleave/ConfirmRequestLeaveViewModel.java

```diff
@@ -91,15 +92,15 @@ public boolean isDetail() {
     }
 
     public boolean isAcceptStatus() {
-        return mRequest.getStatus() == StatusCode.ACCEPT_CODE;
+        return Objects.equals(mRequest.getStatus(), StatusCode.ACCEPT_CODE);
     }
 
     public boolean isPendingStatus() {
-        return mRequest.getStatus() == StatusCode.PENDING_CODE;
+        return Objects.equals(mRequest.getStatus(), StatusCode.PENDING_CODE);
     }
 
     public boolean isRejectStatus() {
-        return mRequest.getStatus() == StatusCode.REJECT_CODE;
+        return Objects.equals(mRequest.getStatus(), StatusCode.REJECT_CODE);
```

Comment:
ERROR: [NewApi] Call requires API level 19 (current min is 16): `java.util.Objects#equals`
 This check scans through all the Android API calls in the application and warns about any calls that are not available on *all* versions targeted by this application (according to its minimum SDK attribute in the manifest).

If you really want to use this API and don't need to support older devices just set the `minSdkVersion` in your `build.gradle` or `AndroidManifest.xml` files.

If your code is *deliberately* accessing newer APIs, and you have ensured (e.g. with conditional execution) that this code will only ever be called on a supported platform, then you can annotate your class or method with the `@TargetApi` annotation specifying the local minimum SDK to apply, such as `@TargetApi(11)`, such that this check considers 11 rather than your manifest file's minimum SDK as the required API level.

If you are deliberately setting `android:` attributes in style definitions, make sure you place this in a `values-vNN` folder in order to avoid running into runtime conflicts on certain devices where manufacturers have added custom attributes whose ids conflict with the new ones on later platforms.

Similarly, you can use tools:targetApi="11" in an XML file to indicate that the element will only be inflated in an adequate context.